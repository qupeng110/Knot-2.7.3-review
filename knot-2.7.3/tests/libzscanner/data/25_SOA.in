$ORIGIN	.
$TTL	1

; OK
@	SOA	@	@	0	0 0 0 0		; The simplest variant
@	SOA	tld.	tld	0	0 0 0 0 	; Absolute and relative dnames.
@	SOA	@	@	4294967295 0 0 0 0	; Maximal serial
@	SOA	@	@	0 4294967295 4294967295 4294967295 4294967295	; Maximal times
@	SOA	@	@	0	1d 1h 1m 1s	; Time units
@	TYPE6	\# 22 00 00 0000000000000000000000000000000000000000	; TYPE + Hexadecimal rdata
@	TYPE6	@	@       0       0 0 0 0		; TYPE
@	SOA	ns.tld. first\.second.tld. (		; Multiline record
	2007120710
	1w2d3h4m5s
	2h
	3m
	4s
)
@	soa	@	@       0       0 0 0 0		; Type in lower-case

; KO
@	SOA
@	SOA						; Empty rdata
@	SOA	\# 0					; Hex empty rdata
@	SOA	@	@	1h	0 0 0 0		; Bad number
@	SOA	@	@	4294967296 0 0 0 0	; Serial overflow
@	SOA	@	@	0 4294967296 0 0 0	; Refresh overflow
@	SOA	@	@	0 0 4294967296 0 0	; Retry overflow
@	SOA	@	@	0 0 0 4294967296 0	; Expire overflow
@	SOA	@	@	0 0 0 0 4294967296	; Minimum overflow
@	SOA	@	@	0 0 0 0 0 x		; Unexpected item
