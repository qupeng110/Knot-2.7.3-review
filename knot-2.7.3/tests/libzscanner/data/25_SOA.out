OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00000000000000000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=03746C640003746C64000000000000000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=0000FFFFFFFF00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=000000000000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=0000000000000001518000000E100000003C00000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00000000000000000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00000000000000000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=026E7303746C64000C66697273742E7365636F6E6403746C640077A23B46000C08A500001C20000000B400000004
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00000000000000000000000000000000000000000000
------
WARNG=ZS_BAD_RDATA
------
WARNG=ZS_BAD_RDATA
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_NUMBER32_OVERFLOW
------
WARNG=ZS_NUMBER32_OVERFLOW
------
WARNG=ZS_NUMBER32_OVERFLOW
------
WARNG=ZS_NUMBER32_OVERFLOW
------
WARNG=ZS_NUMBER32_OVERFLOW
------
WARNG=ZS_BAD_REST
------
